osu.Game/Rulesets/Difficulty/Skills/Skill.cs

```diff
@@ -58,6 +61,21 @@ public void Process(DifficultyHitObject current)
             Previous.Push(current);
         }
 
+        public void TouchProcess(DifficultyHitObject current)
+        {
+            var dummy = prevDummyStrain;
```

Comment:
Words like `dummy` are generally intended to be used for discarded values. This doesn't seem like a discarded value.

Please rename it to something more descriptive.

Reply:
Issue has been resolved as a result of reverting strain calculation change completely.